lib/TransactionRunner.js

```diff
@@ -247,7 +241,7 @@ class TransactionRunner {
     // Transaction skipping (can be modified in hooks). If the input format
     // is OpenAPI 2, non-2xx transactions should be skipped by default.
     let skip = false;
-    if (mediaType.indexOf('swagger') !== -1) {
+    if (transaction.apiDescriptionMediaType.includes('swagger')) {
```

Comment:
Not robust, but will get fixed later: https://github.com/apiaryio/dredd/issues/1257

Reply:
Outside scope of this PR but brings it to my attention, should the same rule apply to OAS3?

Reply:
That's a subject of discussion / pending design. I need to read the spec first to properly understand what's possible to describe with it and what default behavior makes sense for the users.

Eventually, I'd like to unify the behavior across the formats, but that's a long way to go as that's something which depends on the transaction names 'epic' 😬